ionic_user_interface/src/components/VoteButton.vue

```diff
@@ -52,16 +52,27 @@ export default defineComponent({
 
     let waitingForResponse = false;
 
-    // Display hover feedback only if logged in
```

Comment:
Hmmm why is this removed?

Reply:
Cos if he is logged in, he can still get hover feedback and also click. But there will be toast telling him to log in. This should encourage people to sign up instead of thinking the vote button got problem

Reply:
Ok, makes sense